demo/src/test/java/com/example/test/core/keywork/CustomKeyword.java

```diff
@@ -37,10 +39,12 @@ public void openUrl(String baseUrl) throws Exception {
      * Wait for Element displayed and input text
      * @param element
      * @param text
+     * @throws InterruptedException
      */
-    public void sendKeys(WebElement element, String text){
+    public void sendKeys(WebElement element, String text) throws InterruptedException{
         try{
             waitForElementIsDisplayed(element).sendKeys(text);
+            Thread.sleep(3000);

```

Comment:
thread.sleep() should be in completely new function. Do not put it in other function